ledger/ledgertestutil/ledgertestutil.go

```diff
@@ -28,7 +28,7 @@ import (
 
 // TmpLedger crteates ledger on top of temporary database.
 // Returns *ledger.Ledger andh cleanup function.
-func TmpLedger(t *testing.T, dir string) (*ledger.Ledger, func()) {
+func TmpLedger(t testing.TB, dir string) (*ledger.Ledger, func()) {
```

Comment:
Don't see reason for this change? What is the point?

Reply:
we want use helpers in benchmarks

Reply:
Sorry, I was not clear, but what is the point to switch from pointer to not pointer? T -> TB change is perfectly clear.